### PR TITLE
Add update role

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: Dockerfile-canary [hunk 1]
```diff
@@ -1,5 +1,6 @@
 FROM ansibleplaybookbundle/apb-base
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (djzager) [inline Dockerfile-canary]: I think this is supposed to die in each of the files.

File: playbooks/update.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+- name: mediawiki123-apb provision
```
Comment (djzager) [inline playbooks/update.yml]: Should this be `mediawiki123-apb update`?

File: Dockerfile-canary (hunk 1 above)
Comment (djzager) [inline Dockerfile-canary]: I think this is supposed to die in each of the 3 files.

Comment (dymurray) [inline Dockerfile-canary]: This can be removed.

File: Dockerfile-latest [hunk 3]
```diff
@@ -1,5 +1,6 @@
 FROM ansibleplaybookbundle/apb-base:latest
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (dymurray) [inline Dockerfile-latest]: Same as above

File: Dockerfile-nightly [hunk 4]
```diff
@@ -1,5 +1,6 @@
 FROM ansibleplaybookbundle/apb-base:nightly
 
+LABEL "com.redhat.apb.version"="1.0"
```
Comment (dymurray) [inline Dockerfile-nightly]: Same as above

File: playbooks/update.yml (hunk 2 above)
Comment (jmontleon) [reply playbooks/update.yml]: yes. thank you

**Pull request merged**